cmsranking/static/TimeView.js

```diff
@@ -39,6 +45,10 @@ var TimeView = new function () {
     // - 2: current (clock) time
     self.status = 0;
 
+    // Difference in seconds between local time and server time (not
+    // considering timezone offsets).
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'diff_with_server_time' is not in camel case.](https://www.codacy.com/app/s-maggiolo/cms/file/2192640431/issues/source?bid=3167167&fileBranchId=3167181#l50)